modules/cnf-image-based-upgrade-guidelines.adoc

```diff
@@ -0,0 +1,194 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-guide_{context}"]
+= Guidelines for the image-based upgrade
+
+For a successful image-based upgrade, your deployments need to consider certain requirements.
+
+There are different deployment methods in which you can perform the image-based upgrade:
+
+{ztp}:: You utilize the {ztp-first} to deploy and configure your clusters.
+Non-GitOps:: You only use {rh-rhacm-first} to deploy and configure your clusters.
+
+[id="ztp-image-based-upgrade-cluster-validated-software_{context}"]
+== Minimum software version of components
+
+Depending on your deployment method, the image-based upgrade requires the following minimum software versions.
+
+.Minimum software version of components
+[cols=3*, width="80%", options="header"]
+|====
+|Component
+|Software version
+|Required
+
+|{lcao}
+|4.16
+|Yes
+
+|OADP Operator
+|1.3.1
+|Yes
+
+|Managed cluster version
+|4.14.13
+|Yes
+
+|Hub cluster version
+|4.16
+|Yes
+
+|{rh-rhacm}
+|2.10.2
+|Yes
+
+|{ztp} plugin
+|4.16
+|Only for {ztp} deployment method
+
+|{gitops-title}
+|TBD
```

Comment:
This and next TBD entries should be updated with versions or something other than TBD